src/lib/openapi-to-json-schema.ts

```diff
@@ -431,6 +431,15 @@ export default function toJSONSchema(
           } else {
             schema[polyType][idx] = toJSONSchema(item as RMOAS.SchemaObject, polyOptions);
           }
+
+          // Ensure that we don't have any invalid `required` booleans lying around.
+          if (
+            isObject(schema[polyType][idx]) &&
+            'required' in (schema[polyType][idx] as SchemaObject) &&
+            typeof (schema[polyType][idx] as SchemaObject).required === 'boolean'
+          ) {
+            delete (schema[polyType][idx] as SchemaObject).required;
```

Comment:
This'll filter out cases of `required: true` being inside of a schema that looks like this:

```js
oneOf: [
  { type: 'string', required: true}
]
```